client/web/workflow/src/components/WorkflowEditor.vue

```diff
@@ -329,7 +329,7 @@
         <export
           data-test-id="button-export-workflow"
           :workflows="[workflow.workflowID]"
-          :file-name="workflow.handle"
+          :file-name="workflow.meta.name || workflow.handle"
```

Comment:
You should make sure this doesn't include any illegal chars, preferably in the export component
https://stackoverflow.com/questions/42210199/remove-illegal-characters-from-a-file-name-but-leave-spaces